.eslintrc.json

```diff
@@ -64,6 +64,7 @@
   "rules": {
     "import/no-unresolved": "off",
     "import/named": "off",
-    "import/namespace": "off"
+    "import/namespace": "off",
+    "prettier/prettier": "off"
```

Comment:
Is there a reason you've added this?

Reply:
This was because on Windows I get spammed with `Delete `␍`eslint[prettier/prettier](https://github.com/prettier/eslint-plugin-prettier#options)` on every file. I think most people work on MacOS and don't see this. I can revert this change, as I don't know what the proper fix should be.

Reply:
This can be solved via an editorconfig -- https://editorconfig.org. `end_of_line = lf` and `insert_final_newline = true` would both be needed.

Reply:
I couldn't get this to work for me. Can you confirm this is a solution?